providers/class.two-factor-email.php

```diff
@@ -176,6 +176,7 @@ public function authentication_page( $user ) {
 		<p>
 			<label for="authcode"><?php esc_html_e( 'Verification Code:', 'two-factor' ); ?></label>
 			<input type="tel" name="two-factor-email-code" id="authcode" class="input" value="" size="20" pattern="[0-9]*" />
+			<?php submit_button( __( 'Log In', 'two-factor' ) ); ?>
```

Comment:
@wturrell I just noticed that the output of `submit_button()` includes a paragraph tag so this should be outside the wrapping paragraph.